code/game/objects/structures/ghost_role_spawners.dm

```diff
@@ -3,24 +3,23 @@
 //Preserved terrarium/seed vault: Spawns in seed vault structures in lavaland. Ghosts become plantpeople and are advised to begin growing plants in the room near them.
 /obj/effect/mob_spawn/human/seed_vault
 	name = "preserved terrarium"
-	desc = "An ancient machine that seems to be used for storing plant matter. The glass is obstructed by a mat of vines."
+	desc = "Starożytna maszyna, która zdaje się być przeznaczona do przechowywania materii roślinnej. Szyba jest prysłonięta pnączami."
 	mob_name = "a lifebringer"
 	icon = 'icons/obj/lavaland/spawners.dmi'
 	icon_state = "terrarium"
 	density = TRUE
 	roundstart = FALSE
 	death = FALSE
 	mob_species = /datum/species/pod
-	short_desc = "You are a sentient ecosystem, an example of the mastery over life that your creators possessed."
-	flavour_text = "Your masters, benevolent as they were, created uncounted seed vaults and spread them across \
-	the universe to every planet they could chart. You are in one such seed vault. \
-	Your goal is to cultivate and spread life wherever it will go while waiting for contact from your creators. \
-	Estimated time of last contact: Deployment, 5000 millennia ago."
-	assignedrole = "Lifebringer"
+	short_desc = "Jesteś świadomym ekosystemem, przykładem zagadkowego cudu życia, który twoi twórcy poznali."
+	flavour_text = "Twoi mistrzowie w swojej życzliwości stworzyli niezliczone krypty nasion i rozsiali je po całym \
+	wszechświecie, na każdej znanej im planecie. Jesteś w jednej z tych krypt. \
+	Twoim celem jest uprawa i rozsiewanie życia gdziekolwiek się da, w myśl wizji twórców. \
+	Szacowany czas ostatniego kontaktu: 5 tysiącleci temu."
+	assignedrole = "Siewca"
 
 /obj/effect/mob_spawn/human/seed_vault/special(mob/living/new_spawn)
-	var/plant_name = pick("Tomato", "Potato", "Broccoli", "Carrot", "Ambrosia", "Pumpkin", "Ivy", "Kudzu", "Banana", "Moss", "Flower", "Bloom", "Root", "Bark", "Glowshroom", "Petal", "Leaf", \
-	"Venus", "Sprout","Cocoa", "Strawberry", "Citrus", "Oak", "Cactus", "Pepper", "Juniper")
+	var/plant_name = pick("Pomidor Patryk", "Ziemnior", "Brokuł Bartek", "Marchewka Marysia", "Truskaweczka Tosia", "Pieczarka Piotrek", "Gruszka Gosia", "Kudzu", "Bananuch", "Jabłko Antek", "Śliwka Sabina", "Kalafior Krzyś", "Rzodkiewka Żaneta", "Groszek Grześ", "Borówka Basia", "Czosnek Czesio", "Bakłażanek Błażejek")
```

Comment:
To jest piękna lokalizacja. Brakuje mi tylko jednej rzeczy - płci nowego spawnu uzależnionego od nazwy, bo jeśli dobrze widzę to będą spawnować się mężczyżni nazwani "Truskaweczka Tosia" itp. Więc wystarczy lista tupli ze stałą MALE/FEMALE i odpowiednio ustawić H.gender. Dodatkowo, można poszerzyć o `Dąb Dyzio` i `Drzewo Dobromir` z innego gangu.